extensions/quick-access/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quick-access",
+  "title": "Quick Access",
+  "description": "Quick copy, open, reveal the latest files in pinned Finder directories.",
+  "icon": "extension-icon.png",
+  "author": "koinzhang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "pin-directory",
+      "title": "Pin Directory",
+      "icon": "pin-directory.png",
+      "subtitle": "Quick Access",
+      "description": "Pin directories that can be accessed quickly. To use this command, you need to select folders or open a Finder window first.",
+      "mode": "no-view"
+    },
+    {
+      "name": "search-pinned-directory",
+      "title": "Search Pinned Directory",
```

Comment:
```suggestion
      "title": "Search Pinned Directories",
```

Reply:
Yes, the file name and command name have been changed to "Search Pinned Directories".